platform/ios/src/MGLCalloutView.h

```diff
@@ -36,6 +36,18 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, weak) id<MGLCalloutViewDelegate> delegate;
 
 /**
+ A Boolean value indicating whether the callout view should be positioned automatically.
+ The exact position can be adjusted by overriding -[UIView setCenter:]
+ */
+@property (nonatomic, assign) BOOL automaticPositioning;
```

Comment:
Mark these properties as `readonly`, because MGLMapView doesn’t care if the class implements `-setAutomaticPositioning:` – it’ll never call that setter anyways.